pkg/cli/image/manifest/manifest.go

```diff
@@ -404,14 +405,13 @@ func ProcessManifestList(ctx context.Context, srcDigest digest.Digest, srcManife
 			}
 			srcManifests = append(srcManifests, childManifest)
 		}
-
 		switch {
 		case len(srcManifests) == 1 && !keepManifestList:
 			manifestDigest, err := registryclient.ContentDigestForManifest(srcManifests[0], srcDigest.Algorithm())
 			if err != nil {
 				return nil, nil, "", err
 			}
-			klog.V(5).Infof("Used only one manifest from the list %s", srcDigest)
+			klog.Warningf("Chose %s/%s manifest from the list %s\nTo include the manifest list digest: %s, use --keep-manifest-list=true", t.Manifests[0].Platform.OS, t.Manifests[0].Platform.Architecture, srcDigest, srcDigest)
```

Comment:
No need for printing `srcDigest` twice, just leave the first instance.

Reply:
updated